documentdb.go

```diff
@@ -117,6 +119,10 @@ func (c *DocumentDB) ReadDocument(link string, doc interface{}, opts ...CallOpti
 
 // Read sporc by self link
 func (c *DocumentDB) ReadStoredProcedure(link string, opts ...CallOption) (sproc *Sproc, err error) {
+	if c.usesAAD() {
+		return nil, errors.New("cannot perform CRUD operatons on stored procedures while authenticating with Azure AD")
```

Comment:
Can we declare this error once? e.g.

```go
var errAAD = errors.New("cannot perform CRUD operations on stored procedures while authenticating with Azure AD")

/// ...


if c.usesAAD() {
	return nil, errAAD
}
```

There's also a small typo in "operatons => operations".